MAP transformation: Failing tests on Windows

Here are the results on Windows:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.transform.map.internal.MapTransformationServiceTest
[ERROR] Tests run: 7, Failures: 0, Errors: 5, Skipped: 0, Time elapsed: 0.634 s <<< FAILURE! - in org.openhab.transform.map.internal.MapTransformationServiceTest
[ERROR] org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsNotUpdatedIfOldElementMissing  Time elapsed: 0.573 s  <<< ERROR!
java.lang.NullPointerException
        at org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsNotUpdatedIfOldElementMissing(MapTransformationServiceTest.java:126)

[ERROR] org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsRemoved  Time elapsed: 0.006 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'map/doorstatus_de.map' or failed to parse it.
        at org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsRemoved(MapTransformationServiceTest.java:112)

[ERROR] org.openhab.transform.map.internal.MapTransformationServiceTest.testTransformSucceeds  Time elapsed: 0.003 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'map/doorstatus_de.map' or failed to parse it.
        at org.openhab.transform.map.internal.MapTransformationServiceTest.testTransformSucceeds(MapTransformationServiceTest.java:90)

[ERROR] org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsUpdatedIfOldElementPresent  Time elapsed: 0.003 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'map/doorstatus_de.map' or failed to parse it.
        at org.openhab.transform.map.internal.MapTransformationServiceTest.setTransformationConfigurationIsUpdatedIfOldElementPresent(MapTransformationServiceTest.java:142)

[ERROR] org.openhab.transform.map.internal.MapTransformationServiceTest.testTransformSucceedsWithDefault  Time elapsed: 0.003 s  <<< ERROR!
org.openhab.core.transform.TransformationException: Could not find configuration 'map/doorstatus_defaulted.map' or failed to parse it.
        at org.openhab.transform.map.internal.MapTransformationServiceTest.testTransformSucceedsWithDefault(MapTransformationServiceTest.java:102)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   MapTransformationServiceTest.setTransformationConfigurationIsNotUpdatedIfOldElementMissing:126 ▒ NullPointer
[ERROR]   MapTransformationServiceTest.setTransformationConfigurationIsRemoved:112 ▒ Transformation
[ERROR]   MapTransformationServiceTest.setTransformationConfigurationIsUpdatedIfOldElementPresent:142 ▒ Transformation
[ERROR]   MapTransformationServiceTest.testTransformSucceeds:90 ▒ Transformation Could n...
[ERROR]   MapTransformationServiceTest.testTransformSucceedsWithDefault:102 ▒ Transformation
[INFO]
[ERROR] Tests run: 7, Failures: 0, Errors: 5, Skipped: 0
```

--- a/bundles/org.openhab.transform.map/src/test/java/org/openhab/transform/map/internal/MapTransformationServiceTest.java
+++ b/bundles/org.openhab.transform.map/src/test/java/org/openhab/transform/map/internal/MapTransformationServiceTest.java
@@ -13,10 +13,11 @@
 package org.openhab.transform.map.internal;
 
 import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.*;
 
+import java.io.File;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.HashMap;
@@ -49,12 +50,12 @@ public class MapTransformationServiceTest extends JavaTest {
     private static final String SOURCE_CLOSED = "CLOSED";
     private static final String SOURCE_UNKNOWN = "UNKNOWN";
 
-    private static final String NON_DEFAULTED_TRANSFORMATION_DE = "map/doorstatus_de.map";
-    private static final String NON_DEFAULTED_TRANSFORMATION_FR = "map/doorstatus_fr.map";
-    private static final String DEFAULTED_TRANSFORMATION = "map/doorstatus_defaulted.map";
-    private static final String UNKNOWN_TRANSFORMATION = "map/de.map";
+    private static final String NON_DEFAULTED_TRANSFORMATION_DE = "map" + File.separator + "doorstatus_de.map";
+    private static final String NON_DEFAULTED_TRANSFORMATION_FR = "map" + File.separator + "doorstatus_fr.map";
+    private static final String DEFAULTED_TRANSFORMATION = "map" + File.separator + "doorstatus_defaulted.map";
+    private static final String UNKNOWN_TRANSFORMATION = "map" + File.separator + "de.map";
 
-    private static final String SRC_FOLDER = "conf/transform";
+    private static final String SRC_FOLDER = "conf" + File.separator + "transform";
 
     @Mock
     private @NonNullByDefault({}) TransformationConfigurationRegistry transformationConfigurationRegistry;
@@ -67,7 +68,7 @@ public class MapTransformationServiceTest extends JavaTest {
         configurationMap.clear();
         Files.walk(Path.of(SRC_FOLDER)).filter(Files::isRegularFile).forEach(file -> {
             try {
-                String content = new String(Files.readAllBytes(file));
+                String content = new String(Files.readAllBytes(file), StandardCharsets.UTF_8);
                 String uid = Path.of(SRC_FOLDER).relativize(file).toString();
                 TransformationConfiguration transformationConfiguration = new TransformationConfiguration(uid, uid,
                         "map", null, content);
@@ -79,7 +80,7 @@ public class MapTransformationServiceTest extends JavaTest {
         Mockito.when(transformationConfigurationRegistry.get(anyString(), eq(null)))
                 .thenAnswer((Answer<TransformationConfiguration>) invocation -> {
                     Object[] args = invocation.getArguments();
-                    return configurationMap.get((String) args[0]);
+                    return configurationMap.get(args[0]);
                 });
 
         processor = new MapTransformationService(transformationConfigurationRegistry);

